Makefile

```diff
@@ -0,0 +1,115 @@
+# This Makefile should really be a bash script, but I don't know how to make
+# bash scripts that have multiple options, and since we're under deadline
+# pressure I've elected not to learn.
```

Comment:
You would just have multiple bash scripts and choose one to run. Otherwise you would capture a cli argument and run segments based on the argument.